Dockerfile

```diff
@@ -1,11 +1,11 @@
 # Build stage
-FROM golang:1.21@sha256:549dd88a1a53715f177b41ab5fee25f7a376a6bb5322ac7abe263480d9554021 AS builder
+FROM golang:1.22 AS builder
```

Comment:
We we sure we want to go to `1.22`?  I thought we decided to use `1.21`, for now.